ckanext/superset/templates/superset/create-dataset.html

```diff
@@ -28,6 +28,23 @@ <h2>Create CKAN dataset from Superset dataset</h2>
             </select>
           </div>
 
+          <!-- Grup list -->
+          {% set groups_available = h.groups_available('create_dataset') %}
+          <div class="form-group">
+            <label for="ckan_group_id">Groups</label>
+            <div class="checkbox-group">
+              {% for group in groups_available %}
+                <div class="form-check">
+                  <input class="form-check-input" type="checkbox" id="group_{{ group.id }}" name="ckan_group_id" value="{{ group.id }}" 
+                  {% if group.id in selected_groups %}checked{% endif %}>
```

Comment:
```suggestion
                  <input class="form-check-input" type="checkbox" id="group_{{ group.id }}" name="ckan_group_id" value="{{ group.id }}">
```
Nunca va a haber grupos preseleccionados. Siempre vamos a estar creando un dataset nuevo. Hay que sacar tambien `selected_groups` de la vista que llama a este templata